nipype/interfaces/ants/segmentation.py

```diff
@@ -418,34 +417,35 @@ def _format_arg(self, name, trait_spec, value):
             name, trait_spec, value)
 
     def _parse_inputs(self, skip=None):
-        if skip is None:
-            skip = []
-        skip += ['save_bias', 'bias_image']
+        skip = (skip or []) + ['save_bias', 'bias_image']
+        self._out_bias_file = None
+        if self.inputs.save_bias or isdefined(self.inputs.bias_image):
+            bias_image = self.inputs.bias_image
+            if not isdefined(bias_image):
+                bias_image = fname_presuffix(os.path.basename(self.inputs.input_image),
+                                             suffix='_bias', newpath=os.getcwd())
+            self._out_bias_file = bias_image
```

Comment:
I'd put the abspath here, rather than in fname_presuffix, so the bias_image case is covered as well.